client/js/components/procedure/basicSettings/AutoSwitchProcedurePhaseForm.vue

```diff
@@ -119,6 +123,18 @@ export default {
       default: () => ({})
     },
 
+    dataCyEndDate: {
+      type: String,
+      required: false,
+      default: ''
```

Comment:
shouldn't the default be something maningfull like "endDate" ?

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/1860/commits/3a5154131d8f0ea2008098faa275812a8b2d2a87, thnx